fluentui_listitem/src/main/res/values/attrs_list_item.xml

```diff
@@ -39,5 +39,6 @@
             <enum name="medium" value="1" />
             <enum name="large" value="2" />
         </attr>
+        <attr name="fluentui_disabled" format="boolean"/>
```

Comment:
Where is this getting used?

Reply:
Will add the option to use in listitem constructor. Missed it.